digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionCalculator.scala

```diff
@@ -133,22 +138,32 @@ object ProjectSectionCalculator {
     def makeStartCP(projectLink: ProjectLink) = {
       Some(CalibrationPoint(projectLink.linkId, if (projectLink.sideCode == TowardsDigitizing) 0.0 else projectLink.geometryLength, projectLink.startAddrMValue))
     }
+
     def makeEndCP(projectLink: ProjectLink, userDefinedCalibrationPoint: Option[UserDefinedCalibrationPoint]) = {
```

Comment:
Methods makeEndCP and makeEndCPAtStartBorder are equal code wise, should they differ?

Reply:
they should not. . It was duplicated, glad you noticed that